### PR TITLE
Send currency rate to doppler Database

### PR DESCRIPTION
**Pull request by fgchaio**
These changes will allow the Currency Job that runs every day to update doppler's database with the last valid currency for ARS and MXN.

Please have a look at it if you can, and let me know what you think.

TODO: add test

### CODE REVIEW

File: CrossCutting/DopplerSapService/Entities/DopplerCurrencyRate.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using Newtonsoft.Json;
```
Comment (nreal) [inline CrossCutting/DopplerSapService/Entities/DopplerCurrencyRate.cs:1]: this library is not used in the class

File: DopplerCurrencyJob/DopplerCurrencyService/DopplerCurrencyService.cs [hunk 2]
```diff
@@ -74,5 +79,50 @@ public async Task<IList<CurrencyResponse>> GetCurrencyByCode()
...
         }
+
+        public async Task InsertCurrencyIntoDataBase(IList<CurrencyResponse> currencyList)
+        {
+            _logger.LogInformation("Establishing database connection.");
+            try
+            {
+                var parameters = new  List<DopplerCurrencyRate>();
+
+                foreach (CurrencyResponse currency in currencyList)
```
Comment (nreal) [inline DopplerCurrencyJob/DopplerCurrencyService/DopplerCurrencyService.cs:90]: You can use `var ` by `CurrencyResponse`

File: DopplerCurrencyJob/DopplerCurrencyService/DopplerCurrencyService.cs [hunk 3]
```diff
@@ -74,5 +79,50 @@ public async Task<IList<CurrencyResponse>> GetCurrencyByCode()
...
+
+                    parameters.Add(param);
+                }
+
+                var storedProcedure = string.Join("\n", _dopplerCurrencySettings.InsertCurrencyQuery);
+
+                await using var conn = _dbConnectionFactory.GetConnection();
+
+                _logger.LogInformation("Sending SQL sentence to database server.");
+                var result = await conn.ExecuteAsync(storedProcedure,
```
Comment (nreal) [inline DopplerCurrencyJob/DopplerCurrencyService/DopplerCurrencyService.cs:116]: `result ` variable is never used

File: DopplerJobTest/CreateSutCurrencyServiceTests.cs [hunk 4]
```diff
@@ -15,18 +16,21 @@ public static class CreateSutCurrencyServiceTests
...
                 dopplerCurrencySettings,
                 loggerCurrencyService ?? Mock.Of<ILogger<DopplerCurrencyService>>(),
+                
                 timeZoneJobConfigurations ?? new TimeZoneJobConfigurations
                 {
-                    TimeZoneJobs = TimeZoneHelper.GetTimeZoneByOperativeSystem("Argentina Standard Time") 
-                });
+                    TimeZoneJobs = TimeZoneHelper.GetTimeZoneByOperativeSystem("Argentina Standard Time")
+                },
+                dbConnectionFactory);;
```
Comment (nreal) [inline DopplerJobTest/CreateSutCurrencyServiceTests.cs:33]: You can remove `;` additional

File: CrossCutting/DopplerSapService/Entities/DopplerCurrencyRate.cs (hunk 1 above)
Comment (fgchaio) [reply CrossCutting/DopplerSapService/Entities/DopplerCurrencyRate.cs:1]: 🤔 Why did I add that? xD
Thanks!

**Pull request merged**